pkg/plugin/conntrack/_cprog/conntrack.c

```diff
@@ -193,13 +236,22 @@ static __always_inline bool _ct_handle_tcp_connection(struct packet *p, struct c
         p->is_reply = true;
         new_value.flags_seen_rx_dir = p->flags;
         new_value.last_report_rx_dir = now;
+        new_value.bytes_reply_count = p->bytes;
+        new_value.packets_reply_count = 1;
         bpf_map_update_elem(&retina_conntrack, &reverse_key, &new_value, BPF_ANY);
     } else { // Otherwise, the packet is considered as a packet in the send direction.
         p->is_reply = false;
         new_value.flags_seen_tx_dir = p->flags;
         new_value.last_report_tx_dir = now;
+        new_value.bytes_forward_count = p->bytes;
+        new_value.packets_forward_count = 1;
         bpf_map_update_elem(&retina_conntrack, &key, &new_value, BPF_ANY);
     }
+    // Update packet's conntrack metadata.
+    p->conntrack_metadata.bytes_forward_count = new_value.bytes_forward_count;
```

Comment:
I haven't looked, but if these fields are defined in this order in both structs, you can speed this up by doing a memcopy on the byte range they span, instead of one by one mem accesses